gsa/src/web/pages/scanconfigs/editdialog.js

```diff
@@ -327,31 +328,33 @@ class NvtFamily extends React.Component {
           {name}
         </TableData>
         <TableData>
-          <Layout align="end">
+          <Layout align="start">
```

Comment:
Couldn't the align be set on the TableData here? For me the extra Layout seems to be unnecessary.